config/metrics_endpoints.json

```diff
@@ -1,635 +1,1111 @@
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/lightweightm2m-iotagent"
+        ],
+        "docker": [
+            "fiware/lightweightm2m-iotagent"
+        ]
+    },
+    {
+        "enabler": "IoT Agent for LoRaWan",
+        "catalogue": "",
+        "academy": "",
+        "readthedocs": "fiware-lorawan",
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Atos-Research-and-Innovation/IoTagent-LoRaWAN"
+        ],
+        "docker": [
+            "fiware/iotagent-lorawan"
+        ]
+    },
+    {
+        "enabler": "IoT Agent for OPC UA",
+        "catalogue": "",
+        "academy": "",
+        "readthedocs": "iotagent-opcua",
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Engineering-Research-and-Development/iotagent-opcua"
+        ],
+        "docker": [
+            "rdlabengpa/opcuaage"
+        ]
+    },
+    {
+        "enabler": "IoT Agent for SigFox",
+        "catalogue": "",
+        "academy": "IDAS",
+        "readthedocs": "iotagent-sigfox",
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/sigfox-iotagent"
+        ],
+        "docker": [
+            "fiware/sigfox-iotagent"
+        ]
+    },
+    {
+        "enabler": "IoT Agent for Ultralight",
+        "catalogue": "",
+        "academy": "IDAS",
+        "readthedocs": "fiware-iotagent-ul",
+        "helpdesk": "IDAS",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "telefonicaid/iotagent-ul"
+        ],
+        "docker": [
+            "fiware/iotagent-ul"
+        ]
+    },
+    {
+        "enabler": "IoT Broker",
+        "catalogue": "iot-broker",
+        "academy": "IoT Broker",
+        "readthedocs": "fiware-iot-broker",
+        "helpdesk": "IoT Broker",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "Aeronbroker/Aeron",
+            "Aeronbroker/NEConfMan"
+        ],
+        "docker": [
+            "fiware/iotbroker"
+        ]
+    },
+    {
+        "enabler": "IoT Discovery",
+        "catalogue": "iot-discovery",
+        "academy": "IoT Discovery",
+        "readthedocs": "fiware-iot-discovery-ngsi9",
+        "helpdesk": "IoT Discovery",
+        "backlog": "",
+        "jira_workitem": "",
+        "coverall": "",
+        "github": [
+            "UniSurreyIoT/fiware-iot-discovery-s2w",
+            "UniSurreyIoT/fiware-iot-discovery-ngsi9"
+        ],
+        "docker": [
+            "fiware/fiware-iot-discovery-ngsi9"
+        ]
+    },
+    {
+        "enabler": "Keyrock - Identity Manager",
```

Comment:
"enabler": "Keyrock"

Reply:
Fixed in https://github.com/flopezag/fiware-tsc-dashboard/pull/38/commits/55ea9ac2cb5f6bb623a5b2897a9d1e5dede5c585